src/main/java/ch/njol/skript/variables/SerializedVariable.java

```diff
@@ -10,15 +11,23 @@ public class SerializedVariable {
 
 	/**
 	 * The name of the variable.
+	 * <p>
+	 * @deprecated Marked as internal, as this field will soon be private.
 	 */
+	@Deprecated
```

Comment:
Should specify a `since` for these deprecations (can be INSERT VERSION to be filled in on release). I would also point to the replacement methods in the deprecation note.